### DISCUSSION TITLE
New to Sleep and Need data help

### DISCUSSION THREAD

**Discussion by marissakiley** [talmolab/sleap#1176] [Help!] [answered]
Hello! I am new to sleep and am having a lot of trouble figuring out what to do with the data once I download it. I tried downloading H5py, which the site recommended and math labs but I cannot seem to view the data. Any suggestions?

**Comment by roomrys** [comment, marked as answer]
Hi @marissakiley,

Welcome! Some clarifying questions: Which data have you downloaded (file extensions would help)? What are you trying to do with this data (label in SLEAP, do some analysis using one of [our notebooks](https://sleap.ai/notebooks/index.html), etc)?

Thanks,
Liezl

P.S. To get started with SLEAP we recommend [installing from conda package](https://sleap.ai/installation.html#conda-package) and working within your SLEAP conda environment (to activate this environment, you need to run `conda activate sleap` after completing the installation). The SLEAP conda environment should have all dependencies already installed so that you do not need to do anything else.

**Comment by marissakiley** [comment]
Hello!
   After i ran my interference and training, I exported the H5 data (I
believe that is what is called). I am trying to look at the data so I can
see the differences in movement.
    thank you,
          MArissa Pacheco

On Tue, Feb 14, 2023 at 10:30 AM Liezl Maree ***@***.***>
wrote:

> Hi @marissakiley <https://github.com/marissakiley>,
>
> Welcome! Some clarifying questions: Which data have you downloaded? What
> are you trying to do with this data?
>
> Thanks,
> Liezl
>
> P.S. To get started with SLEAP we recommend installing from conda package
> <https://sleap.ai/installation.html#conda-package> and working within
> your SLEAP conda environment (to activate this environment, you need to run conda
> activate sleap after completing the installation).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/talmolab/sleap/discussions/1176#discussioncomment-4972178>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A54GYTDUZCMA3EARQEXEZKLWXOQJTANCNFSM6AAAAAAU3BC32U>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Marissa Pacheco (She/Her)

Class of 2026 | Honors Program |  Health Sciences Major |

Minors in Philosophy and Women Gender Study|

Honors Student Council Officer | Honors Book Club | GEM | GSA

Merrimack College | 315 Turnpike St. | North Andover, MA | 01845

**Comment by roomrys** [comment]
Hi @marissakiley,

Awesome, thanks for the clarification.

I would suggest looking at our [analysis notebook](https://sleap.ai/notebooks/Analysis_examples.html) which could be helpful in setting up your own local notebook (or saving and modifying our colab notebook to meet your needs).

Let me know if you need any help navigating the analysis notebook. I would happy to assist further if needed!

Thanks,
Liezl

P.S. If you would like to work in colab, to get your data into the notebook, you can follow a similar method as we provide in our training/inference notebook (i.e. upload the data to your drive then [mount your drive](https://sleap.ai/notebooks/Training_and_inference_using_Google_Drive.html#mount-your-google-drive) to access the data). Another option (if the data is not too large) is to simply add the following lines to upload your data:
```
from google.colab import files
files.upload()
```

**Comment by marissakiley** [comment]
Thank you! One last question, is there a way to download the file and have
it keep the labels, I was able to download with instances

On Tue, Feb 14, 2023 at 12:34 PM Liezl Maree ***@***.***>
wrote:

> Hi @marissakiley <https://github.com/marissakiley>,
>
> Awesome, thanks for the clarification.
>
> I would suggest looking at our analysis notebook
> <https://sleap.ai/notebooks/Analysis_examples.html> which could be
> helpful in setting up your own local notebook (or saving and modifying our
> colab notebook to meet your needs).
>
> Let me know if you need any help navigating the analysis notebook. I would
> happy to assist further if needed!
>
> Thanks,
> Liezl
>
> P.S. If you would like to work in colab, to get your data into the
> notebook, you can follow a similar method as we provide in our
> training/inference notebook (i.e. upload the data to your drive then mount
> your drive
> <https://sleap.ai/notebooks/Training_and_inference_using_Google_Drive.html#mount-your-google-drive>
> to access the data). Another option (if the data is not too large) is to
> simply add the following lines to upload your data:
>
> from google.colab import files
> files.upload()
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/talmolab/sleap/discussions/1176#discussioncomment-4973585>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A54GYTBFJ2BXXF566D6R6D3WXO63PANCNFSM6AAAAAAU3BC32U>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Marissa Pacheco (She/Her)

Class of 2026 | Honors Program |  Health Sciences Major |

Minors in Philosophy and Women Gender Study|

Honors Student Council Officer | Honors Book Club | GEM | GSA

Merrimack College | 315 Turnpike St. | North Andover, MA | 01845

**Reply by roomrys** [reply]
HI @marissakiley,

I apologize for dropping the ball on this one. Were you able to get it sorted, or do you still need help? I am unsure what you mean by "keep the labels"? Which file are you trying to download?

Thanks,
Liezl
